### ISSUE TITLE
No Lightning strikes

### ISSUE THREAD

**Issue by dan-s-github** [briis/hass-weatherflow2mqtt#246]
### Home Assistant Version?

2023..2

### Add-On or Self Hosted?

HA Supervised (Add-On)

### Weatherflow2MQTT version?

3.2.1

### What happened?

Everything works fine except that there is no lightning strike info available.
new issue as [#195](https://github.com/briis/hass-weatherflow2mqtt/issues/195) was closed and I can not reopen it.
![image](https://github.com/briis/hass-weatherflow2mqtt/assets/20974454/f87ce02c-18be-4975-96f6-02870fa32bd0)
tempest app and website shows them though.
https://tempestwx.com/station/100317/grid
![image](https://github.com/briis/hass-weatherflow2mqtt/assets/20974454/fe029da2-84b5-4e98-b083-e3170ae45790)

don't know if there should be any logs that I would need to check?

**Comment by w1tw0lf** [comment]
Same issue here.

![image](https://github.com/briis/hass-weatherflow2mqtt/assets/55384762/87007c3b-8151-421e-8ec4-30db5c92b0ac)

![image](https://github.com/briis/hass-weatherflow2mqtt/assets/55384762/bc021234-3da8-48fb-b3b5-c5fea29076af)

**Comment by dhover** [comment]
I've read that WeatherFlow also shows the strikes from neighbor stations. I don't know if that is true buit it may be an explanation.

**Comment by w1tw0lf** [comment]
Don't believe it is that @dhover as it did work correctly before.

**Comment by Pel1can111** [comment]
yeah its broken for me as well it seems. I even got the notification from the weatherflow app that there were strikes this morning but mqtt is showing nothing
